barman/cloud.py

```diff
@@ -1140,12 +1160,44 @@ def _abort_multipart_upload(self, upload_metadata, key):
         """
 
     @abstractmethod
+    def _delete_objects_batch(self, paths):
+        """
+        Delete a single batch of objects
+
+        :param List[str] paths:
+        """
+        if len(paths) > self.MAX_DELETE_BATCH_SIZE:
+            raise Exception("Max batch size exceeded")
```

Comment:
Sonarqube picked this up as being too general an exception and it's correct - I'll make this more specific.